sirepo/sim_data/activait.py

```diff
@@ -13,6 +13,8 @@
 class SimData(sirepo.sim_data.SimDataBase):
     @classmethod
     def fixup_old_data(cls, data):
```

Comment:
If we aren't carrying over old simulations then I think all fixup code can be deleted.